docker/Dockerfile

```diff
@@ -1,50 +0,0 @@
-FROM ubuntu:trusty
```

Comment:
Yup, it's in my note on this PR!  Dockerfile and instructions are both broken (and have been for a while).  This PR makes them even more broken, so I fried 'em.